doc/developer.md

```diff
@@ -111,91 +111,68 @@ Build (img:holohub:ngc-v0.6.0-dgpu)...
 
 ### View All Options
 
-Run the command below to view all available launch options in the `dev_container` script:
+Run the command below to view all available launch options in the `holohub` script:
 
 ```sh
-$ ./dev_container help launch
+$ ./holohub run-container --help
 ```
 
 ### Launch a Local Holoscan SDK Container
 
 To use a HoloHub container image built with a local Holoscan SDK container:
 
 ```bash
-$ ./dev_container launch --img holohub:local-sdk-latest --local_sdk_root <path_to_holoscan_sdk>
+$ ./holohub run --img holohub:local-sdk-latest --local-sdk-root <path_to_holoscan_sdk>
 ```
 
 ### Launch a Named HoloHub Container
 
 To launch custom HoloHub container with fully qualified name, e.g. "holohub:ngc-sdk-sample-app"
 
 ```bash
-$ ./dev_container launch --img holohub:ngc-sdk-sample-app
+$ ./holohub run-container --img holohub:ngc-sdk-sample-app
 ```
 
 ### Forward X11 Graphics Over SSH
 
 ```bash
-  ./dev_container launch --ssh_x11
+  ./holohub run-container --ssh-x11
 ```
 
 ### Support Nsight Systems profiling in the HoloHub Container
 
 ```bash
-  ./dev_container launch --nsys_profile
+  ./holohub run-container --nsys-profile
 ```
 
-### Print Verbose Output
-
-```sh
-  ./dev_container launch --verbose
-```
-
-For example, on an x86_64 system with dGPU ```./dev_container launch --verbose``` will print the following values.
-
-```bash
-user@ubuntu-20-04:/media/data/github/holohub$ ./dev_container launch  --verbose
-2023-07-10 18:36:53 $ xhost +local:docker
-non-network local connections being added to access control list
-Launch (HOLOHUB_ROOT: /media/data/github/holohub)...
-Launch (mount_device_opt:  --device /dev/video0:/dev/video0 --device /dev/video1:/dev/video1)...
-Launch (conditional_opt:  -v /usr/lib/libvideomasterhd.so:/usr/lib/libvideomasterhd.so -v /opt/deltacast/videomaster/Include:/usr/local/deltacast/Include)...
-Launch (local_sdk_opt: )...
-Launch (nvidia_icd_json: /usr/share/vulkan/icd.d/nvidia_icd.json)...
-Launch (image: holohub:ngc-v0.6.0-dgpu)...
-....
-```
-
-Please note that the values of some of the variables will vary depending on configured options, iGPU or dGPU, availability of devices for video capture, or other environment factors.
-
 ## Advanced Options for Building Applications
 
 ### View Build Options
 
 ```sh
-./run -h
+./holohub build --help
 ```
 
 ### Local SDK Path
 
 If you have an installation of the Holoscan SDK which is not in a standard path, you may want to provide the root directory of your Holoscan SDK installation.
 
 ```bash
-  ./run build --sdk <path to the Holoscan SDK installation directory>
+  ./holohub build --sdk <path to the Holoscan SDK installation directory>
```

Comment:
for some reason this is not implemented, I think it's the same as`./holohub build --configure-args="-Dholoscan_ROOT=..."`?

Reply:
Looks like [run's --sdk handling](https://github.com/nvidia-holoscan/holohub/blob/main/run#L696) is slightly more advanced and searches for both `public` (internal detail) and root-level Holoscan SDK installations. Not sure how often this is actually used. Seems reasonable to ask the user to point directly to their Holoscan SDK installation, updated inline suggestion to `holoscan_DIR`.